Tapi 2.1.3 fixes

nigel-r-davis opened the pull request:
RIA folder structure changed to align with other releases
Physical route documentation added #628 (in TAPI)
String field definitions updated #622 (in TAPI)

nigel-r-davis committed:
Change RIA folder structure to number based as opposed to version based

nigel-r-davis committed:
TR-547 adjusted for LF and updated with physical route

nigel-r-davis committed:
String fields updated to "any conformant YANG string"

Section explaining added.
Version history updated.

bcjohnso99 commented on ReferenceImplementationAgreements/TR-547/TR-547-TAPI_ReferenceImplementationAgreement_v1.1.docx:
File sizes are different but MS Word compare states that there are no differences in the files (as expected)

nigel-r-davis replied:
I have corrected issues identified and have found and corrected further issues as per description of the change. Please review.

bcjohnso99 reviewed (changes requested):
Formatting changes and typo (sent offline) required for TR-547 1.2

nigel-r-davis committed:
Updates to TR-547

Change from ONF to LF.
Corrections to string fields
Updates to embedded URLs to be formal references
Correction to reference to Alarm and TCA list.
Mentions of 2.1.3 changed to 2.1.x where the comment applied generally to 2.1.3, 4 and 5.
Mentions of 2.3 removed or changed to "future"

nigel-r-davis committed:
Update to reference to TR-527

And one minor tidy up (TAPI to LF TAPI in one reference tag)

nigel-r-davis committed:
Update TR-547-TAPI_ReferenceImplementationAgreement_v1.2.docx

nigel-r-davis commented on ReferenceImplementationAgreements/TR-547/TR-547-TAPI_ReferenceImplementationAgreement_v1.2.docx:
This commit corrects references to TR-548 (which were ONF in some cases and LF in others) and adds TR-548 to the reference list.

bcjohnso99 reviewed (approved):
Looks good
